test_cromwell.sh

```diff
@@ -86,13 +86,17 @@ if [[ -n ${CROMWELL_BRANCH} ]]; then
     git checkout "${CROMWELL_BRANCH}"
     git pull
     echo "Building Cromwell"
-    sbt clean assembly >> ../${ASSEMBLY_LOG} 2>&1
+    sbt clean assembly >> "${ASSEMBLY_LOG}" 2>&1
     cd ..
-    CROMWELL_JAR="cromwell/target/scala-2.11/cromwell-*.jar"
+    CROMWELL_JAR=`find "${RUN_DIR}"/cromwell/target/scala-2.11 -name "cromwell-*.jar"`
 fi
 
 echo "Starting Cromwell, jar is ${CROMWELL_JAR}"
-java "${CONFIG_STRING}" -jar "${CROMWELL_JAR}" server >> "${CROMWELL_LOG}" 2>&1 &
+if [ -n "$CONFIG_STRING" ]; then
```

Comment:
jes uses it, but more generally this script could be used by anything and may or may not have a config. local is just using application.conf